Updates roadmap Q4 2023 (#1365)
publiccodenet/about: organization/roadmap.md

In addition, we will seek organizations that are planning similar large procurements, and help them collaborate around a single shared codebase instead of working in in parallel.

| One year | Five years | Twenty years |
|---|---|---|
| {::nomarkdown}<ul><li>Define and build-out Partnership relations</li><li>International Partner</li><li>Create narratives of collaboration with partners in order to bring more interested parties in</li><li>Provide firesouls with anything they need in order to advocate for us</li></ul>{:/} | {::nomarkdown}<ul><li>Multiple international partners (for instance OS2)</li><li>Examples of networks of partners leading to structural funding</li><li>Example of shared procurement amongst partners</li></ul>{:/} | {::nomarkdown}<ul><li>Vast global ecosystem of partners on many levels of public administration</li><li>Significant percentage of partners heading towards structural funding</li></ul>{:/} |

### Chapters

The Foundation for Public Code is transforming its organizational model to enable international chapters.
There will be a global umbrella organization, along with local chapter organizations.

> The umbrella organization will be an association in which the strategic council, board of directors, and delegates from the chapter organizations work together to define long term priorities.

Reviewer: ```suggestion
The global organization will be an association in which the strategic council, board of directors, and delegates from the chapter organizations work together to define long term priorities.
```

> Local chapters will tailour their work based on the needs of the local public code ecosystem and participating public organizations.

Reviewer: ```suggestion
Local chapters will tailor their work based on the needs of the local public code ecosystem and participating public organizations.
```

> The North America chapter is the first chapter of the Foundation for Public Code.

Reviewer: ```suggestion
The North America chapter is the Foundation for Public Code's first local chapter.
```
